src/DssSpell.t.sol

```diff
@@ -1323,4 +1324,64 @@ contract DssSpellTest is DSTest, DSMath {
         assertTrue(val != address(0));
     }
 
+    address[] deployerAddresses = [
+        0xdDb108893104dE4E1C6d0E47c42237dB4E617ACc,
+        0xDa0FaB05039809e63C5D068c897c3e602fA97457,
+        0xda0fab060e6cc7b1C0AA105d29Bd50D71f036711,
+        0xDA0FaB0700A4389F6E6679aBAb1692B4601ce9bf,
+        0x0048d6225D1F3eA4385627eFDC5B4709Cab4A21c
+    ];
+
+    function checkWards(address addr, string memory contractName) internal {
+        for (uint i = 0; i < deployerAddresses.length; i ++) {
+            (bool ok, bytes memory data) = addr.call(
+                abi.encodeWithSignature("wards(address)", deployerAddresses[i])
+            );
+            if (!ok || data.length != 32) return;
+            uint ward = abi.decode(data, (uint256));
```

Comment:
oops I forgot the `256` to make the variable. Not sure if we want to fix it now and have the CI running from the beginning again.

Reply:
I just fixed. I've added a couple of other tests anyways.